### PR TITLE
fix: Replace snackbars with snackbarshandler

### LINKED ISSUE #2593: Replacing Snackbar.make() in the app with SnackbarHandler.
**Describe the chore**
The snackbars made in the app directly should be made using SnackbarHandler.

**Would you like to work on the issue?**

- [X] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by yashk2000**
Fixed #2593 

Changes: Replace snackbars with snackbarshandler

### CODE REVIEW

**Comment by yashk2000** [comment]
@adityastic please review.

**Comment by iamareebjamal** [comment]
Why?

**Comment by yashk2000** [comment]
We have a separate class which we use for handling snackbars across the app, SnackbarHandler. Some of the snackbars being used currently were just made using snackbar.make, instead of the Handler class for snackbars. So I converted those to SnackbarHandler.

**Comment by iamareebjamal** [comment]
What advantage does SnackbarHandler have?

**Comment by yashk2000** [comment]
@iamareebjamal it was created to have a common class to create all kinds of snackbars except those with actions. We use it to create snackbars with different bottom margins. So all these functions we created in this common class, `SnackbarHandler`, and it is used in almost all places in the app, except the ones which I fixed in this pr.

File: app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java [hunk 1]
```diff
@@ -251,11 +251,12 @@ public void onItemClick(final View childView, final int position) {
...
-            Snackbar.make(
+            Snackbar snackbar =
+                SnackBarHandler.show(
                     findViewById(android.R.id.content),
-                    R.string.Cloudrail_License_key,
-                    Snackbar.LENGTH_SHORT)
-                .show();
+                    getString(R.string.Cloudrail_License_key),
+                    Snackbar.LENGTH_SHORT);
+            snackbar.show();
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java]: This is ridiculous. Why calling show 2 times. It should only be called once

Comment (yashk2000) [reply app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java]: The earlier `.show` is `SnackBarHandler.show()`. This creates a snackbar and returns it. The to show it, we use the second `.show()`.

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java]: Then it should not be called show. It should be create or make

Comment (yashk2000) [reply app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java]: Ok. I'll do that and update the pr.

Comment (yashk2000) [reply app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java]: Done.

File: app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java [hunk 2]
```diff
@@ -187,7 +187,7 @@ public void setUpAdapter(@NotNull RealmQuery<AccountDatabase> accountDetails) {
 
   @Override
   public void showError() {
-    Snackbar snackbar = SnackBarHandler.show(coordinatorLayout, getString(no_account_signed_in));
+    Snackbar snackbar = SnackBarHandler.create(coordinatorLayout, getString(no_account_signed_in));
     snackbar.show();
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java]: No need to save in a variable. Directly call show

**Comment by yashk2000** [comment]
@iamareebjamal please merge.

**Pull request merged**